### PR TITLE
search: uses localized publication date

### PR DESCRIPTION
**Pull request by Glignos**
* Uses smaller font for the publication

### CODE REVIEW

**Comment by Glignos** [comment]
Screenshot 2020-11-02 at 09 23 40

**Comment by Glignos** [comment]
Depends on https://github.com/inveniosoftware/invenio-rdm-records/pull/240

File: invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/search/components.js [hunk 1]
```diff
@@ -29,15 +29,20 @@ export const RDMRecordResultsListItem = ({ result, index }) => {
   );
   const publicationDate = _get(
     result,
-    "ui.publication_date_l10n",
+    "metadata.publication_date_l10n",
```
Comment (lnielsen) [inline invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/search/components.js:32]: Minor: ``publication_date_l10n`` should be in ``ui.publication_date_l10n``. There's something wrong if it is not.

File: invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/search/components.js [hunk 2]
```diff
@@ -29,15 +29,20 @@ export const RDMRecordResultsListItem = ({ result, index }) => {
...
     "No publication date found."
   );
+  const updatedDate = _get(
+    result,
+    "updated_date_l10n",
```
Comment (lnielsen) [inline invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/search/components.js:37]: Minor: It should live inside ``ui`` key. If it doesnt' there's something wrong in the REST API seriazlier.

```
{
  "ui": {
    "publication_date_l10n": "...",
    "updated_l10n": "...",
  }
}

Comment (lnielsen) [reply invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/search/components.js:37]: See also nveniosoftware/invenio-rdm-records#203

**Review by lnielsen — APPROVED** [review]

**Pull request merged**